go.mod

```diff
@@ -27,4 +27,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220405142115-a332d4046e57
```

Comment:
Reminder.

Reply:
addressed in [d8b5e30](https://github.com/codeready-toolchain/host-operator/pull/622/commits/d8b5e301ec13ef8f865584a09fd65cf89e208126)